<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ctrl_r">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>61</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Ctrl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>_
-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_31">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>G
g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>ENTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_44">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>M
m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="capslock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Caps Lock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_29">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>D
d</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_35">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>L
l</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_40">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>C
c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_22">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>O
o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_37">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;
'</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>#
3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Tab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_20">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>U
u</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_19">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Y
y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_33">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>J
j</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_42">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>B
b</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_r">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Win</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_15">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>W
w</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>)
0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_45">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;
,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_47">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>?
/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="space">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>361</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>@
2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_l">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SHIPT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_30">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>F
f</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_13">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>+
=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="win_l">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Win</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_28">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>S
s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_18">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>T
t</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>*
8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ctrl_l">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Ctrl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backspace">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Back space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_21">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>I
i</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_25">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>}
]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_41">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>V
v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_16">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>E
e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_36">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>:
;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_38">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Z
z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copy">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_43">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>N
n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_14">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Q
q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_46">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;
.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_17">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>R
r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>^
6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_32">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>H
h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_10">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>(
9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_27">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>A
a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp; 
 7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_24">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>{
[</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>$
4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>!
1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_23">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>P
p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_r">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>290</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SHIPT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kor">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Kor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_34">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>K
k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_39">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>X
x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>~
`</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>%
5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_26">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>91</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>| 
 \</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
